knowhere/feder/IVFFlat.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2019-2020 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License
+
+#pragma once
+
+#include <memory>
+#include <tuple>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+#include <nlohmann/json.hpp>
+
+#include "knowhere/feder/IDSet.h"
+
+namespace knowhere::feder::ivfflat {
+
+///////////////////////////////////////////////////////////////////////////////
+// index view
+struct ClusterInfo {
+    int64_t id_;
+    std::vector<int64_t> node_ids_;
+    std::vector<float> centroid_vec_;
+
+    ClusterInfo() = default;
```

Comment:
Need explicit here

Reply:
cpplint reports error if mark default constructor as explicit

Reply:
Fair, it is somehow redundant.